selfdrive/car/honda/carstate.py

```diff
@@ -267,7 +267,7 @@ def update(self, cp, cp_cam, cp_body):
 
   def get_can_parser(self, CP):
     messages = get_can_messages(CP, self.gearbox_msg)
-    return CANParser(DBC[CP.carFingerprint]["pt"], messages, get_pt_bus(CP.carFingerprint))
+    return CANParser(DBC[CP.carFingerprint]["pt"], messages, CanBus(CP).pt)
```

Comment:
Can you open a new PR that adds this CanBus class for Bosch/Nidec?

Reply:
sure, can do